appsensor-core/src/main/java/org/owasp/appsensor/core/Attack.java

```diff
@@ -30,14 +29,14 @@
  * @author John Melton (jtmelton@gmail.com) http://www.jtmelton.com/
  */
 @Entity
-public class Attack implements Serializable {
+public class Attack implements IAppsensorEntity {
 
 	private static final long serialVersionUID = 7231666413877649836L;
 
 	@Id
-	@Column
+	@Column(columnDefinition = "integer")
```

Comment:
@MaikJaek I've not seen this technique before of setting the definition to an integer type for the DB, and using a string in the actual class. Can you point me to some documentation on this? Also, what's the reason this is necessary?